src/components/typography/typography.tsx

```diff
@@ -197,3 +197,10 @@ export const LeadboxText = styled.p`
   margin: 0 auto;
   max-width: 320px;
 `;
+
+export const TextfieldLabel = styled.span`
```

Comment:
As discussed in the meeting just now, I suggest to add one more level of abstraction to all of our typography configurations. In Figma this configuration is called `Text XS`:
![image](https://user-images.githubusercontent.com/7970458/142613627-36078a26-a74d-4117-976d-d52ba297bddb.png)

Would you mind moving this as a local style to the component and using it there. We will start to refactor all the typography settings into a single file once we introduced a proper separation between old and new components.

So the suggestion would be, inside your component file:
```ts
const textXS = css`
  // font settings (except color)
`;

const StyledLabel = styled.span`
  ${textXS}
  // other properties
`;
```

Take a look at [this comment](https://github.com/styled-components/styled-components/issues/773#issuecomment-299667585) if something is unclear.